Makefile

```diff
@@ -76,12 +76,12 @@ assume-role:
 	rm .assume_role_json
 
 drop-tables: upgrade-pip install-backend
-	pip install alembic
+	pip install 'alembic==1.9.4'
```

Comment:
Exactly, on March 6th they released 1.10.1 (data.all pipeline was working fine on March 5th) which included some regression fix that seems related to our issue.

Reply:
Hi @kukushking I have tested it in a deployment and the migration completes successfully (both in the validate migration step and in the actual migration).

What I was thinking is that instead of pinning the version directly in the CodeBuild command, we could add alembic as a backend package in `backend/requirements.txt` so that it is treated as any other package. The only downside that I see is that alembic is not a strictly requirement of the backend compute resources. What is your opinion?

Reply:
I think it's all right, no strong opinion on this one. As long as there is a comment specifying why it's there. Also considering migration script might be called outside CB.